lib/l10n/app_es.arb

```diff
@@ -0,0 +1,539 @@
+  "settingsVideoQuickActionEditorTitle": "Acciones rápidas",
+
+  "settingsSubtitleThemeTile": "Subtítulos",
+  "settingsSubtitleThemeTitle": "Subtítulos",
+  "settingsSubtitleThemeSample": "Esto es un ejemplo.",
+  "settingsSubtitleThemeTextAlignmentTile": "Alineación de texto",
+  "settingsSubtitleThemeTextAlignmentTitle": "Alineación de texto",
+  "settingsSubtitleThemeTextSize": "Tamaño de texto",
+  "settingsSubtitleThemeShowOutline": "Mostrar contorno y sombra",
+  "settingsSubtitleThemeTextColor": "Color de texto",
+  "settingsSubtitleThemeTextOpacity": "Opacidad de texto",
+  "settingsSubtitleThemeBackgroundColor": "Color de fondo",
+  "settingsSubtitleThemeBackgroundOpacity": "Opacidad de fondo",
+  "settingsSubtitleThemeTextAlignmentLeft": "Izquierda",
+  "settingsSubtitleThemeTextAlignmentCenter": "Centro",
+  "settingsSubtitleThemeTextAlignmentRight": "Derecha",
+
+  "settingsSectionPrivacy": "Privacidad",
+  "settingsAllowInstalledAppAccess": "Permita el acceso a lista de aplicaciones",
+  "settingsAllowInstalledAppAccessSubtitle": "Usado para mejorar los álbumes mostrados",
+  "settingsAllowErrorReporting": "Permitir reporte de errores anónimo",
+  "settingsSaveSearchHistory": "Guardar historial de búsqueda",
+
+  "settingsHiddenItemsTile": "Elementos ocultos",
+  "settingsHiddenItemsTitle": "Elementos ocultos",
+
+  "settingsHiddenFiltersTitle": "Filtros",
+  "settingsHiddenFiltersBanner": "Fotos y videos que concuerden con los filtros no aparecerán en su colección.",
+  "settingsHiddenFiltersEmpty": "Sin filtros",
+
+  "settingsHiddenPathsTitle": "Ubicaciones ocultas",
+  "settingsHiddenPathsBanner": "Fotos y videos que se encuentren en estos directorios y cualquiera de sus subdirectorios no aparecerán en su colección.",
+  "addPathTooltip": "Añadir ubicación",
+
+  "settingsStorageAccessTile": "Acceso al almacenamiento",
+  "settingsStorageAccessTitle": "Acceso al almacenamiento",
+  "settingsStorageAccessBanner": "Algunos directorios requieren un permiso de acceso explícito para que sea posible modificar los archivos que contienen. Puede revisar los directorios con permiso aquí.",
+  "settingsStorageAccessEmpty": "Sin permisos de acceso",
+  "settingsStorageAccessRevokeTooltip": "Revocar",
+
+  "settingsSectionAccessibility": "Accesibilidad",
+  "settingsRemoveAnimationsTile": "Remover animaciones",
+  "settingsRemoveAnimationsTitle": "Remove animaciones",
+  "settingsTimeToTakeActionTile": "Hora de entrar en acción",
+  "settingsTimeToTakeActionTitle": "Hora de entrar en acción",
+
+  "settingsSectionLanguage": "Idioma y formatos",
+  "settingsLanguage": "Idioma",
+  "settingsCoordinateFormatTile": "Formato de coordenadas",
+  "settingsCoordinateFormatTitle": "Formato de coordenadas",
+  "settingsUnitSystemTile": "Unidades",
+  "settingsUnitSystemTitle": "Unidades",
+
+  "statsPageTitle": "Stats",
+  "statsWithGps": "{count, plural, =1{1 elemento con ubicación} otros{{count} elementos con ubicación}}",
```

Comment:
`otros` should stay as `other`. it is part of the syntax to handle single/plural, just like `=1`